libnmstate/state.py

```diff
@@ -155,6 +225,76 @@ def merge_interfaces(self, other_state):
             dict_update(other_state.interfaces[name], self.interfaces[name])
             self._ifaces_state[name] = other_state.interfaces[name]
 
+    def merge_route_config(self, other):
+        """
+        Merge routes from other to self.
+        If interface does exists in self, create an empty one.
+        Delete route entries if 'state: absent'.
+        If certain property missing when deleting, treat it as wildcard.
```

Comment:
`If a property is mising, treat it as a wildcard.` IMHO there are articles and the predicate missing.